src/modules/sub-items/components/sub-items-list/sub.items.list.component.js

```diff
@@ -6,15 +6,35 @@ import GridViewComponent from '../grid-view/grid.view.component.js';
 
 import './css/sub.items.list.component.css';
 
+const translationsOfSortNames = {
+    'LocationPriority': 'priority',
+    'ContentName': 'name',
+    'DateModified': 'date'
+}
+
 const views = {
-    table: TableViewComponent,
-    grid: GridViewComponent,
+    table: {
+        component: TableViewComponent,
+        propsTransformer: (props) => {
+
+            props.sortKey = translationsOfSortNames[Object.keys(props.sortClauses)[0]] || 'name';
```

Comment:
The whole logic of sorting is inside the `table.view.component.js` maybe it would be better to have this there in the constructor? We would safe calculation on each rerender and will not have dummy callback in `grid`.